timemachine/cpp/src/neighborlist.cu

```diff
@@ -209,7 +228,8 @@ void Neighborlist<RealType>::compute_block_bounds_device(
 	const int N, // Number of atoms
 	const int D, // Box dimensions
 	const double *d_coords, // [N*3]
-        const double *d_box, // [D*3]
+    const double *d_box, // [D*3]
```

Comment:
indentation? (tabs?)

Reply:
What is our preference for the C++? Spaces or tabs?

Reply:
I sure as hell hope everything's spaces...

Reply:
Looks like I changed this from tabs to spaces, others are still tabs. Will swap them to spaces

Reply:
Please do! This was embarassing